wasm/patch-artifacts-enable-wormhole.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 echo "Patching wasm artifacts to enable wormhole via APIs that compile and instantiate wasm module"
-sed -i.bak 's/var result = WebAssembly.instantiateStreaming(response, info);/var result = WebAssembly.instantiateStreaming(response, info, {simdWormhole:true});/g' wasm/bergamot-translator-worker.js
-sed -i.bak 's/return WebAssembly.instantiate(binary, info);/return WebAssembly.instantiate(binary, info, {simdWormhole:true});/g' wasm/bergamot-translator-worker.js
-sed -i.bak 's/var module = new WebAssembly.Module(bytes);/var module = new WebAssembly.Module(bytes, {simdWormhole:true});/g' wasm/bergamot-translator-worker.js
+sed -i.bak 's/WebAssembly.instantiateStreaming[[:space:]]*([[:space:]]*response[[:space:]]*,[[:space:]]*info[[:space:]]*)/WebAssembly.instantiateStreaming(response, info, {simdWormhole:true})/g' wasm/bergamot-translator-worker.js
+sed -i.bak 's/WebAssembly.instantiate[[:space:]]*([[:space:]]*binary[[:space:]]*,[[:space:]]*info[[:space:]]*)/WebAssembly.instantiate(binary, info, {simdWormhole:true})/g' wasm/bergamot-translator-worker.js
+sed -i.bak 's/WebAssembly.Module[[:space:]]*([[:space:]]*bytes[[:space:]]*)/WebAssembly.Module(bytes, {simdWormhole:true})/g' wasm/bergamot-translator-worker.js
```

Comment:
Feels like a wrong mechanism to accomplish some source transformation, no? Is this our only option? 

If yes, please make this script take a `$WASM_BUILD_DIR` option as a first positional argument, default to current directory. Assume this directory contains the WASM built file here. I don't want to be changing this source all the time over [here](https://github.com/jerinphilip/bergamot-translator-build/blob/a1c386a3ee8005aed78c10fb5df1d95777c7a6f3/Makefile#L83-L86).

Reply:
> Feels like a wrong mechanism to accomplish some source transformation, no? Is this our only option?

Unfortunately, we don't have any other option of doing it.

> If yes, please make this script take a `$WASM_BUILD_DIR` option as a first positional argument, default to current directory. Assume this directory contains the WASM built files where sed is attempted. I don't want to be changing this source all the time over [here](https://github.com/jerinphilip/bergamot-translator-build/blob/a1c386a3ee8005aed78c10fb5df1d95777c7a6f3/Makefile#L83-L86).

@jerinphilip I am fine with making this change but not as a part of this PR because the aim of this PR is different. It will cause edits in github workflow files as well. I will create another PR focusing on just this change.